tools/webinstall/install.sh

```diff
@@ -199,6 +199,27 @@ get_user_response() {
     _RETVAL="$_gur_read_answer"
 }
 
+# prompt_user_yes_no asks the user a question which must be answered with yes or no. Returns a boolean value.
+# $1: question
+# $2: default answer
+# Returns:
+# _RETVAL: true for "yes", false for "no"
+prompt_user_yes_no() {
+  while true;
+  do
+      get_user_response "$1" "$2"
+      _answer="$_RETVAL"
+
+      if printf "%s" "$_answer" | "$GREP" -q -E "$_NO_ANS"; then
+          return 1
+      elif printf "%s" "$_answer" | "$GREP" -q -E "$_YES_ANS"; then
+          return 0
```

Comment:
I think this does not catch default cases?

see `_NO_ANS_DEFAULT`/`_YES_ANS_DEFAULT`

Reply:
I don't think those are actually needed. The default case (an empty string) is already handled with the second argument of get_user_response. The only thing that makes the `_NO_ANS_DEFAULT/_YES_ANS_DEFAULT `regexes different is that they match an empty string as well, so if I'm not mistaken that check was duplicated in the original script as well.

Reply:
They match also empty strings and only newlines

I'll try it like this now and see if it works

Reply:
Tried it out now

Seems to work, nice